### PR TITLE
UHF-8010: After-school activities listing search

### PR DESCRIPTION
**Pull request by Jussiles**
# [UHF-8010](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8010)

## What was done

- Add new Views After-school activities search based on Daycare search View
- Add new Paragraph for the View to block to be shown on Landing page and basic page
- Add needed theming, styles and translations

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8010-after-school-listing-search`
  * `make fresh`
* Run `make drush-cr`
* Run `make shell`
* Run `drush locale-check && drush locale-update && drush cr`

## How to test

* [x] Add After-school activities search block to Landing page content area or Basic page lower content area.
* [x] Check that the listing looks good and search and pagination works. (It should look as same as Daycare search https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus/varhaiskasvatus/varhaiskasvatus-paivakodissa/etsi-kunnallisia-paivakoteja but it should only have address filter). If listing is empty for some reason, go to this page: https://helfi-kasko.docker.so/en/childhood-and-education/admin/content/integrations/tpr-unit?search=iltap%C3%A4iv%C3%A4toiminta&langcode=All and add Iltapäivätoiminta to Unit type field for some TPR entities.
* [x] Check that mobile view works.
* [x] Check Swedish and English translations work.
* [ ] Check that code follows our standards

## Designers review

* [X] This PR does not need designers review

[UHF-8010]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8010?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by teroelonen — CHANGES_REQUESTED** [review]
There seems to be at least for me something wrong with the sorting by address. I searched with Mannerheimintie 1 and the results seem to be organized by distance but the distance is quite far from Helsinki.

The  address on the card is also hard to read because there is no comma or new line between the street address and the postal code. Not sure if it is actually an issue with this PR or the cards in general.

Then there was one question about one of the code changes in hdbt_subtheme.theme 🙏

File: public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme [hunk 1]
```diff
@@ -301,7 +301,7 @@ function hdbt_subtheme_preprocess_field(&$variables) {
     'zh' => t('Chinese', [], ['context' => 'Language of instruction']),
   ];
 
-  foreach ($variables['items'] as $delta => &$item) {
+  foreach ($variables['items'] as &$item) {
```
Comment (teroelonen) [inline public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme:304]: Why is this change done?

Comment (Jussiles) [reply public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme:304]: Delta wasn't used and php linter complained about it so I removed in the name of cleaning.

Comment (teroelonen) [reply public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme:304]: Perfect!

**Comment by Jussiles** [comment]
Bugs that Tero found are already in production. Those will be fixed in other ticket/PR.

**Pull request merged**